### PR TITLE
Create trip model

### PR DESCRIPTION
**Pull request by g0g05arui**
PR for issue [#19]

In this PR:
- changed app name from `com.android.sample` to `com.android.voyageur`
- added the data classes for `Location` and `Trip`
- added firebase dependencies ( needed for `Timestamp` class )

Note:
The activity model is not yet finalised, so when instantiating this class just use an empty array for the `activities` field

### CODE REVIEW

File: app/src/main/java/com/android/voyageur/model/trip/Trip.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import com.android.voyageur.model.location.Location
+import com.google.firebase.Timestamp
+
+data class Trip(
+    val id: String,
+    val creator: String,
+    val participants: Array<String>,
+    val description: String,
+    val name: String,
+    val location: Location,
```
Comment (ioanajianu) [inline app/src/main/java/com/android/voyageur/model/trip/Trip.kt:12]: What if someone is doing a trip to multiple locations? For example multiple cities within the same country? Or multiple countries?
Let's maybe make this `val locations: Array<Location>` to support these cases, wdyt?

**Review by ioanajianu — CHANGES_REQUESTED** [review]
I had just a comment about the Trip class, otherwise looks great, thanks!

File: app/src/main/java/com/android/voyageur/model/trip/Trip.kt (hunk 1 above)
Comment (g0g05arui) [reply app/src/main/java/com/android/voyageur/model/trip/Trip.kt]: Yes you are right, I made it an array.

**Review by ioanajianu — APPROVED** [review]
LGTM!

**Pull request merged**